src/main/java/com/asu/cse545/group12/dao/AuthorizationDaoImpl.java

```diff
@@ -171,4 +171,20 @@ public int updateRow(Authorization authorization) {
 		tx.commit();
 		return authorization.getAuthorizationId();
 	}
+
+	@Override
+	public List<Authorization> getAuthorizednotifications(int fromuser,
+			int touser) {
+		Session session = sessionfactory.openSession(); 
+		String status="APPROVED";
+		String reqType= "view profile";
+		Query query= session.createQuery("from authorization where AUTHORIZED_BY_USERID=:touser and AUTHORIZED_TO_USERID=:fromuser  and REQUEST_TYPE=:reqType");
+		query.setParameter("fromuser", fromuser);
+		query.setParameter("touser", touser);
+		//query.setParameter("status", status);
+		query.setParameter("reqType", reqType);
+		List<Authorization> authlist = query.list();
+		System.out.println(authlist);
```

Comment:
These fields are not editable.

On Thu, Oct 22, 2015 at 12:38 PM, jaswitha vankineni <
vankineni.jaswitha@gmail.com> wrote:

> Yes , i think i forgot to add in that class. I will remove the sysout.I
> use that as it is convenient for me.
>
> On Thu, Oct 22, 2015 at 12:35 PM, Nagarjuna Myla <notifications@github.com
> > wrote:
>
>> In src/main/java/com/asu/cse545/group12/dao/AuthorizationDaoImpl.java
>> <https://github.com/chasethenag420/sbs/pull/87#discussion_r42795436>:
>>
>> > @@ -171,4 +171,20 @@ public int updateRow(Authorization authorization) {
>> >  		tx.commit();
>> >  		return authorization.getAuthorizationId();
>> >  	}
>> > +
>> > +	@Override
>> > +	public List<Authorization> getAuthorizednotifications(int fromuser,
>> > +			int touser) {
>> > +		Session session = sessionfactory.openSession();
>> > +		String status="APPROVED";
>> > +		String reqType= "view profile";
>> > +		Query query= session.createQuery("from authorization where AUTHORIZED_BY_USERID=:touser and AUTHORIZED_TO_USERID=:fromuser  and REQUEST_TYPE=:reqType");
>> > +		query.setParameter("fromuser", fromuser);
>> > +		query.setParameter("touser", touser);
>> > +		//query.setParameter("status", status);
>> > +		query.setParameter("reqType", reqType);
>> > +		List<Authorization> authlist = query.list();
>> > +		System.out.println(authlist);
>>
>> Why dont you use logger class instead of system.out so we can disable it
>> later on.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/chasethenag420/sbs/pull/87/files#r42795436>.
>>
>
>